From 8cbd395af77651f551537c11b15e9e20d744675c Mon Sep 17 00:00:00 2001
From: kboeluek <kboeluek@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 24 Jan 2014 13:27:49 +0000
Subject: [PATCH 0442/1000] MANTIS 0025026 Mantis 0025026 Report process
 counters to logging service

-added an method to PlatformIF interface. RA_ and CA_ attributes
will be logged through this method.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@643 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index 5ee6e7eb..fe0a6f76 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -28,7 +28,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "1.4";
+    const string interface_version = "1.5";
 
     /** Data type for text lists. */
 
@@ -194,7 +194,7 @@ module PlatformManagement
       LC_CAPTURE_PRINTER,
       LC_CAPTURE_READER,
       LC_DISPENSER_PRINTER,
-	  LC_DISPENSER_READER,
+      LC_DISPENSER_READER,
       LC_PRINTER_FEEDER,
       LC_CONVEYOR_FEEDER,
       LC_DISK,
@@ -235,6 +235,7 @@ module PlatformManagement
         void logCounter(in LoggingCounterData lcd);
         void logCounterFailure(in string unitId, in LoggingClass unitClass, in long numberOfFailed);
         void updateMediaCount(in string compRef, in long mediaCount);
+        void logCounterSuccess(in string unitId, in LoggingClass unitClass, in long numberOfSuccesfullDocs);
         
         /** Checks the status of the token passed by the application.
          *  @param token - The application token
-- 
2.41.0.windows.1

